tests/bud/use-layers/Dockerfile.multistage-copy

```diff
@@ -0,0 +1,9 @@
+FROM alpine AS uuid
+COPY uuid /src
+
+FROM alpine AS date
+COPY date /src
+
+FROM alpine
+COPY --from=uuid /src/data /uuid
+COPY --from=date /src/data /date
```

Comment:
Would be nice to have some checks here to make sure the files got in the right places.

Reply:
FWIW, you could do something like #1182 

```
ctr=$(buildah --debug=false from --signature-policy ${TESTSDIR}/policy.json ${target})
  mnt=$(buildah --debug=false mount ${ctr})
  run test -e $mnt/usr/local/bin/composer
```
which is something I stole from a @nalind suggestion several reviews ago.

Reply:
oh okay, will add such a check.